activate.sh

```diff
@@ -46,3 +46,14 @@ if ! [ -f /root/.ssh/id_rsa ]; then
 fi
 [ -d /var/lib/misc/ssh-public-key  ] || mkdir -p /var/lib/misc/ssh-public-key
 cp /root/.ssh/id_rsa.pub /var/lib/misc/ssh-public-key
+
+# Connect the salt-minion running in the administration controller node to the local salt-master
+# instance that is running in a container
+cat <<EOF > /etc/salt/grains
+roles:
+- admin
+EOF
+echo "master: localhost" > /etc/salt/minion.d/minion.conf
+echo "id: admin" > /etc/salt/minion.d/minion_id.conf
+
+systemctl enable salt-minion
```

Comment:
No, this is executed at the very end of the YaST installation, we don't have a `salt-master` yet running. After this script is executed, the machine will get rebooted.